docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java

```diff
@@ -48,12 +49,12 @@
 import java.util.List;
 import java.util.Map;
 
-import static org.elasticsearch.common.xcontent.ConstructingObjectParser.constructorArg;
-
 import static java.util.Collections.emptyMap;
 import static java.util.Collections.singletonList;
 import static java.util.Collections.singletonMap;
+import static org.elasticsearch.common.xcontent.ConstructingObjectParser.constructorArg;
 
+@TimeoutSuite(millis = 30 * TimeUnits.MINUTE)
```

Comment:
Can you leave a comment here summarizing our current thinking on this timeout value.

Reply:
Something like "The default 20 minutes timeout is not always enough, please do not increase further than 30 minutes without analyzing what makes these tests so slow" ?

Reply:
Yeah, something like that indeed. 😄